docs/node-operator.md

```diff
@@ -130,8 +143,14 @@ Once your node from step 1 is fully synced then you can stop the node by pressin
 **Running from docker**
 
 ```sh
-docker run bitcountry/bitcountry-node:5f860f4 --chain tewai --validator --name 'your node name' --bootnodes /ip4/13.239.118.231/tcp/30344/p2p/12D3KooW9rDqyS5S5F6oGHYsmFjSdZdX6HAbTD88rPfxYfoXJdNU --telemetry-url 'wss://telemetry.polkadot.io/submit/ 0' 
+docker run --restart=unless-stopped --network=host -v /tewaiDb/bitcountry-node:/bitcountry-db bitcountry/bitcountry-node:5f860f4 -d /bitcountry-db --chain tewai --validator --name 'your node name' --bootnodes /ip4/13.239.118.231/tcp/30344/p2p/12D3KooW9rDqyS5S5F6oGHYsmFjSdZdX6HAbTD88rPfxYfoXJdNU --telemetry-url 'wss://telemetry.polkadot.io/submit/ 0' 
```

Comment:
Could you add -p for port mapping between host machine and docker container? This will be beneficial to query the session key.

Reply:
> Could you add -p for port mapping between host machine and docker container? This will be beneficial to query the session key.

It's user's preferences, everyone has different port available to map.

Reply:
I van add port mapping, but to had access to RPC localhost ports from outside the docker use should use --network=host -- for sure it is better to run docker with port mapping, more secure, but we should begin to answer users how they can get to rpc from docker and more of this.

While running compilled binary version do the same as --network=host here -- maps to host ports